crmsh/corosync.py

```diff
@@ -282,9 +282,15 @@ def valid_attr(self):
         if not utils.valid_port(self.port):
             raise ValueError("invalid qdevice port range(1024 - 65535)")
 
+        if self.algo not in ["ffsplit", "lms"]:
```

Comment:
It's a minor issue, so it's not that important. :wink: 
Code-wise, it's perfectly fine to use a list. However, I normally use a tuple here when I have a constant array of values and want to emphasize that this is something that isn't modified:

```suggestion
        if self.algo not in ("ffsplit", "lms"):
```

Reply:
Changed, thanks!